litmus-portal/frontend/src/views/CreateWorkflow/VerifyCommit/index.tsx

```diff
@@ -281,7 +281,9 @@ const VerifyCommit: React.FC<VerifyCommitProps> = ({
         onClose={handleClose}
         width="60%"
         modalActions={
-          <ButtonOutlined onClick={handleClose}>&#x2715;</ButtonOutlined>
```

Comment:
Can we use icon from Litmus as per Figma rather than Unicode character being passed as children to button component? cc: @arkajyotiMukherjee

Reply:
This is essentially a colour fix PR @whitetiger1399 cc: @arkajyotiMukherjee All modals are using such Unicode values, file changes will be 50+ if all changes are made in this PR itself.

Reply:
I don't believe it'll cross that many files... unicode is fine if it looks exactly like the design

Reply:
I have not updated any code just added a class file @whitetiger1399